internal/provider/integration_aws_serverless_resource.go

```diff
@@ -279,20 +300,38 @@ func (r *integrationAwsServerlessResource) Schema(ctx context.Context, req resou
 								Optional:            true,
 								ElementType:         types.StringType,
 							},
+							"exclude_instance_ids_filter": schema.ListAttribute{
+								MarkdownDescription: "List of instance IDs to exclude.",
+								Optional:            true,
+								ElementType:         types.StringType,
+							},
+							"exclude_regions_filter": schema.ListAttribute{
+								MarkdownDescription: "List of regions to exclude.",
+								Optional:            true,
+								ElementType:         types.StringType,
+							},
+							"exclude_tags_filter": schema.MapAttribute{
+								MarkdownDescription: "Excluded Tags filter.",
+								Optional:            true,
+								ElementType:         types.StringType,
+							},
 							"ebs_volume_scan": schema.BoolAttribute{
 								MarkdownDescription: "Enable EBS volume scan.",
 								Optional:            true,
 							},
 							"ebs_scan_options": schema.SingleNestedAttribute{
-								Required: true,
+								Required:           true,
```

Comment:
If this argument is deprecated, can we not make it a required field?

Reply:
good point!

Reply:
Once removed, I'll approve!